tests/bulk.rs

```diff
@@ -0,0 +1,110 @@
+use futures::{lock::Mutex, AsyncRead, AsyncWrite};
```

Comment:
Missing tests:

- write a wrong number of columns
- write data which is of wrong type

Reply:
this type of test doesn't need a connection, I will prefer to write unit tests.

For the first one, add one test in token_row.rs: wrong_number_of_columns_will_fail;
for the second one, there is already one in column_data.rs: wrong_number_of_columns_will_fail